packages/clarity-visualize/src/layout.ts

```diff
@@ -41,7 +41,18 @@ export function markup(event: Layout.DomEvent, iframe: HTMLIFrameElement): void
     let doc = iframe.contentDocument;
     for (let node of data) {
         let parent = element(node.parent);
-        let next = element(node.next);
+        let pivot = element(node.previous);
+        let insert = insertAfter;
+
+        // For backward compatibility. Until v0.4.4 we used the next element to determine right ordering within DOM
+        // Starting with v0.4.5, we moved away from the next element and instead starting sending previous element
```

Comment:
Why does this change (using previous instead of next) help? Add that to comment here.

Reply:
Sure, thanks.